### PR TITLE
Es filetemp

### PR DESCRIPTION
**Pull request by jvelilla**

### CODE REVIEW

File: Src/library/base/elks/kernel/plain_text_file.e [hunk 1]
```diff
@@ -20,7 +20,7 @@ create
 	make, make_with_name, make_with_path,
 	make_open_read, make_open_write, make_open_append,
 	make_open_read_write, make_create_read_write,
-	make_open_read_append
+	make_open_read_append, make_temporary_open, make_temporary_open_with_prefix
```
Comment (jocelyn) [inline Src/library/base/elks/kernel/plain_text_file.e]: I would put them on new line, to "group" the temporary creation proc.

File: Src/library/base/elks/kernel/raw_file.e [hunk 2]
```diff
@@ -20,7 +20,7 @@ create
 	make, make_with_name, make_with_path,
 	make_open_read, make_open_write, make_open_append,
 	make_open_read_write, make_create_read_write,
-	make_open_read_append
+	make_open_read_append, make_temporary_open, make_temporary_open_with_prefix
```
Comment (jocelyn) [inline Src/library/base/elks/kernel/raw_file.e]: I would group the temporary* creation proc on the next line.

File: Src/library/base/ise/kernel/dotnet/file.e [hunk 3]
```diff
@@ -165,6 +165,22 @@ feature -- Initialization
 			open_append: is_open_append
 		end
 
+	make_temporary_open
```
Comment (jocelyn) [inline Src/library/base/ise/kernel/dotnet/file.e:168]: what about  make_temporary_open_with_prefix ?
is it available for dotnet?

File: Src/library/base/ise/kernel/dotnet/plain_text_file.e [hunk 4]
```diff
@@ -24,7 +24,7 @@ create
 	make, make_with_path,
 	make_with_name, make_open_read, make_open_write, make_open_append,
 	make_open_read_write, make_create_read_write,
-	make_open_read_append
+	make_open_read_append, make_temporary_open
```
Comment (jocelyn) [inline Src/library/base/ise/kernel/dotnet/plain_text_file.e:27]: what about  make_temporary_open_with_prefix ?

File: Src/library/base/ise/kernel/dotnet/raw_file.e [hunk 5]
```diff
@@ -26,7 +26,7 @@ create
 	make, make_with_path,
 	make_with_name, make_open_read, make_open_write, make_open_append,
 	make_open_read_write, make_create_read_write,
-	make_open_read_append
+	make_open_read_append, make_temporary_open
```
Comment (jocelyn) [inline Src/library/base/ise/kernel/dotnet/raw_file.e:29]: what about make_temporary_open_with_prefix ?

File: Src/library/base/elks/kernel/file.e [hunk 6]
```diff
@@ -171,6 +171,40 @@ feature -- Initialization
 			open_append: is_open_append
 		end
 
+	make_temporary_open
+			-- Create a unique temporary file name.
```
Comment (jocelyn) [inline Src/library/base/elks/kernel/file.e]: replace the dot by a comma

Comment (jocelyn) [inline Src/library/base/elks/kernel/file.e]: why not "Create Current file in read/write mode with unique temporary file name." ?
or something similar.

File: Src/library/base/elks/kernel/file.e [hunk 7]
```diff
@@ -171,6 +171,40 @@ feature -- Initialization
...
+		do
+			make_temporary_open_with_prefix("tfn")
+		ensure
+			exists: exists
+			open_read: is_open_read
+			open_append: is_open_write
+		end
+
+	make_temporary_open_with_prefix (a_prefix: READABLE_STRING_GENERAL)
+			-- Create a unique temporary file name with prefix `a_prefix'.
```
Comment (jocelyn) [inline Src/library/base/elks/kernel/file.e]: replace dot by comma

File: Src/library/base/elks/kernel/plain_text_file.e (hunk 1 above)
Comment (jvelilla) [reply Src/library/base/elks/kernel/plain_text_file.e]: ok good

File: Src/library/base/elks/kernel/raw_file.e (hunk 2 above)
Comment (jvelilla) [reply Src/library/base/elks/kernel/raw_file.e]: ok good.

File: Src/library/base/ise/kernel/dotnet/file.e (hunk 3 above)
Comment (jvelilla) [reply Src/library/base/ise/kernel/dotnet/file.e:168]: Not sure need to check.

File: Src/library/base/elks/kernel/file.e (hunk 6 above)
Comment (jvelilla) [reply Src/library/base/elks/kernel/file.e]: The file is open and the mode is read_write. Am I missing something?

Comment (jvelilla) [reply Src/library/base/elks/kernel/file.e]: oh you mean the description of the feature. yes need to be updated.

Comment (jvelilla) [reply Src/library/base/elks/kernel/file.e]: What about this?
`Create a file object with a unique temporary file name with read/write mode'

**Pull request closed without merging**